### PR TITLE
Add dialog component

### PR DESCRIPTION
**Pull request by annmedvid**

### CODE REVIEW

File: app/components/item-details/reset.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+button, figure
+{
+    all: unset;
+}
+
+input[type=radio]
```
Comment (shvaikalesh) [inline app/components/item-details/reset.css]: We probably don't want all radios on all pages to be hidden.

File: app/components/item-details/fonts.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@font-face
+{
+    font-family: "GothamProMedium";
+    src: url("fonts/GothamProMedium.ttf") format("truetype");
```
Comment (shvaikalesh) [inline app/components/item-details/fonts.css]: Let's make url root-based: `/fonts`

File: app/components/item-details/fonts.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@font-face
+{
+    font-family: "GothamProMedium";
```
Comment (shvaikalesh) [inline app/components/item-details/fonts.css]: Let's not include font weight properties (like `medium` or `bold`) in family names: it does not scale well. Better specify `font-size` instead.

File: app/components/item-details/variables.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    --mercury: #e9e9e9;
+    --shark: #252a2d;
+    --gray-chateau: #a4a7a9;
+    --chambray: #384d9d;
+    --d-ocre: #ffafab;
+    --d-grey: #d2ccc3;
+    --abbey: #484a4b;
+    --gothampromed: "GothamProMedium" , sans-serif;
+    --soursesanproreg: "SourceSansProRegular" , sans-serif;
+    --size-widht-full: 450px;
```
Comment (shvaikalesh) [inline app/components/item-details/variables.css]: Oops, typo here.

File: app/components/item-details/reset.css [hunk 5]
```diff
@@ -3,7 +3,7 @@ button, figure
     all: unset;
 }
 
-input[type=radio]
+.item-details input[type=radio]
```
Comment (shvaikalesh) [inline app/components/item-details/reset.css]: Let's move this decl out of `reset.css`.

File: app/components/item-details/variables.css (hunk 4 above)
Comment (annmedvid) [reply app/components/item-details/variables.css]: Like this? And is it all?
```css
    --gothampro: "GothamProMedium" , sans-serif;
    --soursesanpro: "SourceSansProRegular" , sans-serif;
```

**Pull request merged**